### PR TITLE
finished task 11

### PR DESCRIPTION
**Pull request by JC-Ellis**
1. Completed task 11 "happy path" and "sad path"
2. Updated some descriptions to better reflect actual behavior
3. Refactored some tests in app-errors.test for better readability

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -224,65 +224,110 @@ describe("GET /api/users", () => {
...
+              expect(typeof article.title).toBe("string");
+              expect(typeof article.article_id).toBe("number");
+              expect(typeof article.created_at).toBe("string");
+              expect(typeof article.votes).toBe("number");
+              expect(typeof article.article_img_url).toBe("string");
+              expect(typeof article.comment_count).toBe("string");
+            });
+          });
       });
+    });
```
Comment (nc-jo-watson) [inline __tests__/app.test.js:331]: A missing test here: `200: Responds with an empty array when request query is for a topic that doesn't have any associated articles`.

I think in your db this is the topic `paper` 🤔
